### PR TITLE
Master1

### PR DESCRIPTION
**Pull request by alin4ik15**

### CODE REVIEW

**Comment by alin4ik15** [comment]
Видишь?

File: src/task19.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+public class task19 {
+
+    public static void main(String[] args) {
+
+        String s = "How are you?";
```
Comment (helenhva) [inline src/task19.java]: "How are  , you?"
Print  - 5 word ...
Something is wrong

File: src/task21.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+public class task21 {
+
+        public static void main(String[] args) {
+            String s = "How are you?";
```
Comment (helenhva) [inline src/task21.java]: "How are , you?"
Exception in thread "main" java.lang.StringIndexOutOfBoundsException: String index out of range: -1

File: src/task22/Time.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            hour = h;
+            minut = m;
+            second = s;
+        }
+
+        public Time(int allsec) {
+            second = allsec;
+        }
+
+        public int returnAllSecond() {
```
Comment (helenhva) [inline src/task22/Time.java]: Method name does not conform to convention

**Review by helenhva — COMMENTED** [review]
Please correct bugs, and also use packages

**Pull request closed without merging**